### PR TITLE
[release_tool] move onert_version.sh into release_tool

### PR DESCRIPTION
**Pull request by glistening**
onert_version/onert_version.sh is moved under release_tool.

ONE-DCO-1.0-Signed-off-by: Sanggyu Lee <sg5.lee@samsung.com>

### CODE REVIEW

File: tools/release_tool/README.md [hunk 1]
```diff
@@ -37,3 +42,28 @@ $ ./git_release.sh --tag v1.1 --commitish c024e85d0ce6cb1ed2fbc66f1a9c1c2814da75
...
+```
+$ ./onert_version.sh -h
+Usage: onert_version.sh version
+Update or show onert version information
+```
+
+## Options
+```
+	-h   show this help
+	-s   set onert  version
```
Comment (mhs4670go) [inline tools/release_tool/README.md:62]:
```suggestion
-h   show this help
-s   set onert  version
```

File: tools/release_tool/README.md [hunk 2]
```diff
@@ -37,3 +42,28 @@ $ ./git_release.sh --tag v1.1 --commitish c024e85d0ce6cb1ed2fbc66f1a9c1c2814da75
...
+```
+
+## Examples
+```
+	onert_version.sh           => show current onert version
+	onert_version.sh -s 1.6.0  => set onert version info in all sources
```
Comment (mhs4670go) [inline tools/release_tool/README.md:68]: ditto

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by chunseoklee — DISMISSED** [review]

**Comment by glistening** [comment]
@chunseoklee, @mhs4670go I've fixed a typo.

`onert_release.sh` → `onert_version.sh` from `Content`.

**Review by chunseoklee — APPROVED** [review]

**Comment by mhs4670go** [comment]
@nnfw-bot test android-runtime-builder

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**